dask/highlevelgraph.py

```diff
@@ -78,6 +78,36 @@ def get_output_keys(self) -> Set:
         """
         return self.keys()
 
+    def expand_annotations(self, string_keys=True) -> Mapping[str, Any]:
```

Comment:
It looks like string_keys is always True.  If so, perhaps we should consider removing this parameter for simplicity's sake.

Reply:
Removed